pymc3/tests/test_distributions.py

```diff
@@ -2707,6 +2723,65 @@ def test_bound():
         BoundPoissonPositionalArgs = Bound(Poisson, upper=6)("x", 2.0)
 
 
+class TestBoundedContinuous:
```

Comment:
Tests look like they go in the right direction. I'll check them more carefully, just wanted to mention a nitpick: I would pass the missing lower / upper explicitly as `None`, to make them easier to read.